Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM cypress/included:11.2.0
+FROM cypress/included:12.17.1 as builder
 
-RUN apt-get -y install gosu locales && \
-    sed -i '/de_CH.UTF-8/s/^# //g' /etc/locale.gen && \
-    locale-gen
+RUN apt update
+RUN apt install curl locales -y 
```

Comment:
Best practice would be to do that in one command and also get rid of the packages afterwards in order to keep this layer small:
```suggestion
RUN set -eux; \
    apt-get update && apt-get install -y --no-install-recommends \
    curl \
    locales \
    && rm -rf /var/lib/apt/lists/*

```

Reply:
That's great 😍 thank you so much. Then I think we no longer need the multi-stage building, right?